fiona/_meta.pyx

```diff
@@ -0,0 +1,43 @@
+
+include "gdal.pxi"
+
+from fiona._err cimport  exc_wrap_pointer
+from fiona.compat import strencode
+from fiona._shim cimport gdal_open_vector
+from fiona.env import require_gdal_version
+import logging
+
+# This is required otherwise GDALGetDriverByName returns NULL for GDAL 1.x
+from fiona._shim cimport *
```

Comment:
I don't know why GDALGetDriverByName returns NULL here but not in ogrext without this import. As metadata is mostly only available for gdal >= 2.0, this is not that important, but I find it still strange.